### PR TITLE
Fix shortcodes in “Encrypting Secret Data at Rest”

### PR DESCRIPTION
**Pull request by sftim**
Fix shortcodes in https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/ [[preview](https://deploy-preview-20922--kubernetes-io-master-staging.netlify.app/docs/tasks/administer-cluster/encrypt-data/)] following [Hugo upgrade](https://github.com/kubernetes/website/pull/19907).

### CODE REVIEW

**Comment by sftim** [comment]
/retitle Fix shortcodes in “Encrypting Secret Data at Rest”

**Comment by kbhawkey** [comment]
@sftim, Would you _squash_ your commits? Thanks!

**Comment by kbhawkey** [comment]
@sftim, Looks like there was only a couple of areas that actually required changes.
The rest of the changes are additional cleanup and new note shortcodes.

**Comment by sftim** [comment]
I've done this as two commits to keep the changes logically separate.

**Comment by kbhawkey** [comment]
> 
> 
> I've done this as two commits to keep the changes logically separate.

@sftim Yes, I understand. However, the changes are on one page and interelated.
**If I make an exception for this PR, I will feel obliged to make the same exception for every PR that does not squash the commits.**
What do you think?

**Comment by sftim** [comment]
> If I make an exception for this PR, I will feel obliged to make the same exception for every PR that does not squash the commits.

When changes are logically separate I usually recommend separating them. Both in this project and elsewhere, I've actually asked people to “unsquash” a commit: separating logically separate changes into 2 different commits. It helps with reverts, for example, and with understanding the history from `git annotate`.

This is different from the situation where you have an initial attempt then several fixups following code review. Once the set of changes look right, it's definitely appropriate to squash the fixups.

In summary: I think it's not only OK to sometimes have PRs with more than 1 commit, it can actually be good thing.
